plugin/trino-mongodb/pom.xml

```diff
@@ -133,6 +138,19 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>units</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.vaadin.external.google</groupId>
+            <artifactId>android-json</artifactId>
+            <version>0.0.20131108.vaadin1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why do we need this library ?

Reply:
It's required by JSONAssert. I remove it and used manual assert instead.